reaper/bucket.go

```diff
@@ -0,0 +1,114 @@
+package reaper
```

Comment:
**hint**: Incidental - healthier to have each reaper in a separate file.  Bucket reaper updated work with Bucket model objects.  Introduces expiration (like File).  No other logic changes.